opc/scripts/setup/claude_integration.py

```
@@ -931,6 +964,32 @@ def uninstall_opc_integration(
+    if "zsh" in shell:
+        configs_to_clean.append(_os.path.expanduser("~/.zshrc"))
+    elif "bash" in shell:
+        configs_to_clean.append(_os.path.expanduser("~/.bashrc"))
+    elif "fish" in shell:
+        fish_config = _os.path.expanduser("~/.config/fish/config.fish")
+        if _os.path.exists(fish_config):
+            configs_to_clean.append(fish_config)
+
+    for config_path in configs_to_clean:
+        try:
+            with open(config_path, "r") as f:
```

Ruff (0.15.9)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)